packages/code-studio/src/main/EmptyDashboard.tsx

```diff
@@ -1,16 +1,9 @@
-import React, { MouseEvent } from 'react';
+import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { Button } from '@deephaven/components';
 import { vsArrowUp } from '@deephaven/icons';
 import './EmptyDashboard.scss';
 
-export interface EmptyDashboardProps {
-  onAutoFillClick?: (event: MouseEvent) => void;
```

Comment:
We're just getting rid of the autofill behaviour entirely now? Why?

Reply:
I forgot to commit stashed changes. Fixed